### PR TITLE
Improving autotune instructions for Phase A, plus overall clarity and readability

### PR DESCRIPTION
**Pull request by danamlewis**

### CODE REVIEW

File: docs/docs/Customize-Iterate/autotune.md [hunk 1]
```diff
@@ -1,55 +1,73 @@
...
-* Install jq: `sudo apt-get install jq`
-* Make two copies of your profile.json, one to be the starting point for autotune, and one to provide the pump baseline for enforcing the min/max limits: `cd ~/myopenaps/settings/ && cp profile.json autotune.json && cp profile.json pumpprofile.json`
-* Run `oref0-autotune --dir=~/myopenaps --ns-host=https://mynightscout.azurewebsites.net --start-date=YYYY-MM-DD` (obviously, sub in your NS url and the start date you want to start with. Try 1 day first before moving on to 1 week and 1 month to better troubleshoot).
+* Run this command to create a `newdirectory` and copy over the profile and pump settings files: `mkdir -p ~/newdirectory/settings && cd ~/newdirectory/settings && cp ~/myopenaps/settings/profile.json ~/newdirectory/settings/autotune.json && cp ~/myopenaps/settings/pumpprofile.json ~/newdirectory/settings/pumpprofile.json`
```
Comment (scottleibrand) [inline docs/docs/Customize-Iterate/autotune.md]: the `cd ~/newdirectory/settings && ` here isn't needed.

File: docs/docs/Customize-Iterate/autotune.md [hunk 2]
```diff
@@ -1,55 +1,73 @@
...
 
-**Note:** Once you run autotune, the rig will start using the autotune settings as part of the closed loop. The difference between the manual method and the automatic method (below) is that the manual method won't change the settings until you run autotune manually again; the automatic method changes the settings nightly.  
+**To run Autotune manually:** 
+* `oref0-autotune --dir=~/newdirectory --ns-host=https://mynightscout.azurewebsites.net --start-date=YYYY-MM-DD` 
+  * obviously, sub in your NS url and the start date you want to start with
+  * If you change your pump settings, you will need to re-copy your pump settings back into `newdirectory`
 
-#### Phase B: Running Autotune automatically in OpenAPS 
+**Note:** If you did this correctly in your `newdirectory`, settings will not be used by OpenAPS. You will need to `cat-autotune` inside of `newdirectory` to see your autotune recommendations, and it will only run when you manually run it. The recommended behavior is to run Autotune inside of your OpenAPS directory, per Phase B, which is the default and will automatically run every night and have OpenAPS use the settings from Autotune automatically.
```
Comment (scottleibrand) [inline docs/docs/Customize-Iterate/autotune.md]: cat-autotune does `cd ${myopenaps}/autotune`, so it won't show the results from `newdirectory`.  you can `cd~/newdirectory/autotune && cat autotune_recommendations.log` though.

File: docs/docs/Customize-Iterate/autotune.md (hunk 1 above)
Comment (danamlewis) [reply docs/docs/Customize-Iterate/autotune.md]: resolved

File: docs/docs/Customize-Iterate/autotune.md (hunk 2 above)
Comment (danamlewis) [reply docs/docs/Customize-Iterate/autotune.md]: resolved

**Comment by cluckj** [comment]
Tested the instructions, and it remains sandboxed in ~/newdirectory/

**Pull request merged**